tests/libcxx/skipped_tests.txt

```diff
@@ -1594,3 +1594,16 @@ namespace\addressable_functions.sh.cpp
 strings\basic.string\string.capacity\shrink_to_fit.explicit_instantiation.sh.cpp
 thread\thread.condition\thread.condition.condvarany\wait_terminates.sh.cpp
 utilities\format\format.arguments\format.arg.store\make_format_args.sh.cpp
+
+# REQUIRES: locale.en_US.UTF-8

```

Comment:
I'll add `(the Windows version on Contest VMs doesn't always understand ".UTF-8")` here. This is plenty clear enough for folks who've been through this mess twice, but I think inscrutable to new maintainers.

Reply:
Agreed, thank you!